### DISCUSSION TITLE
OTBR_NO_AUTO_ATTACH

### DISCUSSION THREAD

**Discussion by kmfnjtze8888** [openthread/openthread#7772] [Q&A] [answered]
Hello,

On the ot-br-posix in `/etc/default/otbr-agent`, there is an `OTBR_NO_AUTO_ATTACH`. I couldn't find anywhere on what it's supposed to do. Under what circumstances will I need to set this to 1 or 0?

**Comment by jwhui** [comment, marked as answer]
By default, OTBR will attempt to start Thread networking on start - see [ThreadHelper::TryResumeNetwork](https://github.com/openthread/ot-br-posix/blob/9ddfb7a1a39f9c0c7d84b634a278703f3093694b/src/utils/thread_helper.cpp#L468). Setting `OTBR_NO_AUTH_ATTACH=1` will disable this behavior and require the user/client to explicitly enable Thread networking.
